README.md

```diff
@@ -299,6 +299,8 @@ K9s uses aliases to navigate most K8s resources.
     headless: false
     # Set to true to hide K9s crumbs. Default false
     crumbsless: false
+    # Makes config file read only. Runtime config changes only stored in memory. Default false
+    lockConfig: false
```

Comment:
I think we don't need an extra config for this. We should write a log and ignore this error silently. @derailed what do you think?

Reply:
@torjue Thank you for the clarification on #1642!! 
I agree with @slimus here as it might be better to turn off persistence with a simple cli arg for this use case. To boot a persistent flag to make the config transient is almost comical... 
Also can you please add some docs in the readme. Thank you Torjus!!

Reply:
@torjue Sorry for the confusion. You can still use your manual override and drop the config flag state so it doesn't get marshalled/unmarshalled during the load/save stage an exported accessor for this would suffice. Since all we want here is the manual override only and not having this state bleed thru the config file. Does this make sense?